src/v2/spec.yaml

```diff
@@ -4335,7 +4418,7 @@ paths:
     get:
       tags:
         - Cast
-      summary: Fetch cast for a given hash or Warpcast URL
+      summary: Fetch cast by hash or URL
```

Comment:
Do we need "fetch" in the title? The "By Eth or Sol addresses" didnt have it? Lets be consistent and remove "fetch" or similar verbs from everywhere while we are at it